packages/sdk-codegen-scripts/src/prettify.ts

```diff
@@ -35,18 +35,15 @@ const prettierTs: prettier.Options = {
   semi: true,
   singleQuote: true,
   trailingComma: 'es5',
-  arrowParens: 'always',
+  arrowParens: 'avoid',
```

Comment:
the primary cause of all the file reformatting in this PR. Required for compatibility with internal versions of the SDK.